regression-tests/mixed-initialization-safety-3-contract-violation.cpp2

```diff
@@ -26,4 +26,5 @@ fill: (
 print_decorated: (x:_) = std::cout << ">> [" << x << "]\n";
 
 // for test determinism, force "fill" branch
-flip_a_coin: () -> bool = false;
+// the standard mandates that std::mt19937()() == 3499211612
```

Comment:
[Explanation] This is true because the standard defines the default seed.

Reply:
Calling `std::rand()` multiple times will return different results, but calling `std::mt19937()()` will always return the same result, so this changes meaning (if the flip function were called multiple times)?